tasks/go.py

```diff
@@ -21,7 +21,7 @@
 from .utils import get_build_flags
 
 # List of modules to ignore when running lint
-MODULE_WHITELIST = [
+MODULE_ALLOWLIST = [
```

Comment:
I asked here https://github.com/DataDog/datadog-agent/pull/8846#discussion_r689297476 for a separate PR to do this; so it is already being taken care of (I would remove it from here to keep things tidy)

Reply:
If you folks want to keep the rename in this PR, I think logistically it makes more sense, @derekwbrown @mx-psi?

Reply:
I think here makes more sense FWIW.

Reply:
Let's do it here then, the only thing missing is fixing the print on line 114